Changelog.md

```diff
@@ -1,21 +1,53 @@
 ## Gazebo Math 7.x
 
-### Gazebo Math 7.1.0
+### Gazebo Math 7.2.0 (2023-06-14)
+
+1. Add tests to increase test coverage
+    * [Pull request #533](https://github.com/gazebosim/gz-math/pull/533)
+
+1. Forward ports
+    * [Pull request #530](https://github.com/gazebosim/gz-math/pull/530)
+    * [Pull request #526](https://github.com/gazebosim/gz-math/pull/526)
+    * [Pull request #522](https://github.com/gazebosim/gz-math/pull/522)
+    * [Pull request #520](https://github.com/gazebosim/gz-math/pull/520)
+
+1. Disable pybind11 on windows by default
+    * [Pull request #529](https://github.com/gazebosim/gz-math/pull/529)
+
+1. Add option to skip pybind11 and SWIG
+    * [Pull request #480](https://github.com/gazebosim/gz-math/pull/480)
+
+1. Custom PID error rate
+    * [Pull request #525](https://github.com/gazebosim/gz-math/pull/525)
+
+1. Garden bazel support
+    * [Pull request #523](https://github.com/gazebosim/gz-math/pull/523)
+
+1. Rename COPYING to LICENSE
+    * [Pull request #521](https://github.com/gazebosim/gz-math/pull/521)
+
+1. Infrastructure
```

Comment:
maybe use a more descriptive entry? `CI workflow: use checkout v3`

Reply:
We've been putting just "Infrastructure" for things like these since it's not interesting for most users (See item 33 in the 7.0.0 changelog).